src/types.ts

```diff
@@ -30,6 +30,8 @@ import { ItemRenderer, ItemRendererStatus } from "./ItemRenderer";
  * @property - The default direction value when direction is not set in the render option. <ko>render옵션에서 direction을 미설정시의 기본 방향값.</ko>
  * @property - You can set the ItemRenderer directly externally. <ko>외부에서 직접 ItemRenderer를 설정할 수 있다.</ko>
  * @property - You can set the ContainerManager를 directly externally. <ko>외부에서 직접 ContainerManager를 설정할 수 있다.</ko>
+ * @property - Whether to get the size as offsetWidth, offsetHeight. Set to true if transform is applied to the container. If false, get the size through getBoundingClientRect. (default: true) <ko>사이즈를 offsetWidth, offsetHeight로 가져올지 여부.
+container에 transform이 적용되어 있다면 true로 설정해라. false면 getBoundingClientRect를 통해 사이즈를 가져온다. (default: true)</ko>
```

Comment:
The API description alone tells you when to use this part. I think that's a good explanation. If you think more about it, I think it will be easier for users to judge whether or not transform is used as described, rather than whether or not offset should be used. So how about changing the option name to something like "usedTransform"?

Reply:
@mixed 

There is already an option called `useTransform`.

Also, useOffset is an option that can be used even if transform is not used.

Reply:
useRoundedSize